app/menu.js

```diff
@@ -22,6 +23,12 @@ const template = [{
     click () {
       ResolveIPNS.create(app)
     }
+  }, {
+    label: 'PublishToIPNS',
+    accelerator: 'CmdOrCtrl+F',
```

Comment:
I would invert Resolve (aka Find) and Publish:

- CmdOrCTRL+F => Resolve
- CmdOrCTRL+E => Publush

Does it make more sense? otherwise lets find something else. CMD+P? (used for Print, but what do we have to print? :P )